docs/installation.md

```diff
@@ -1,27 +1,30 @@
 # Installation
 
-
-This is a 3.2 Django project, running on MySQL, and using RabbitMQ as a message queue, and Redis for caching.
-
-It is tested on OS X and linux systems.
-
-To install it, create a virtual environment, and install the required packages.
-
-```
-make venv
-. venv/bin/activate
-pip install pipenv
-pipenv install
-```
-
-### Set up the environment
-
-By default, pipenv loads the contents of `.env` for any commands like opening a shell with `pipenv shell`, or running one off commands with `pipenv run command_name`.
-
-Copy the sample environemnt file at `.env.sample` to `.env`, and fill in the necessary environment variables:
-
-In production the admin_portal project relies on object storage for storing static files, but in development you may not need this.
-
-
-
-
+This installation shows you how to setup the Admin Portal of The Green Web Foundation.<br>
+In order to setup this project, we recommend following the guides from top to bottom. Skipping some topics might result in missing crucial steps for the project to work.
+
+---
+## Gitpod
+### Why Gitpod
+To speed up and simplify the installation process, we use Gitpod. <br> We strongly recommend using this tool.
```

Comment:
Would you change this to say:

> This is the main supported approach for setting up a development environment. You can use other approaches, but we may not be able to provide as much support.

This based around the idea of a Golden path - there's a blessed approach we take, but people are free to use their own tools as long as they realise we're not able to dedicate the same resources to helping there.

More below.

https://engineering.atspotify.com/2020/08/how-we-use-golden-paths-to-solve-fragmentation-in-our-software-ecosystem/